### PR TITLE
Mm compiler flags

### PR DESCRIPTION
**Pull request by mlm2299**
Adds cmc script for the user to compile with.

### CODE REVIEW

**Comment by joshuazweig** [comment]
http://www.thegeekstuff.com/2012/05/encrypt-bash-shell-script/

**Comment by joshuazweig** [comment]
What is the cmc.sh target for?

**Comment by mlm2299** [comment]
Nothing just haven't pushed yet -- one sec

**Comment by joshuazweig** [comment]
@mlm2299 pull before any additional changes, i pushed a commit

**Comment by mlm2299** [comment]
@joshuazweig yeah we good -- can you take a look at the path statement since it doesn't seem to work on my machine at least

**Comment by joshuazweig** [comment]
@mlm2299 It cant find path bc the path isn't actually updatin. echo $PATH after the make all, it does not persist

File: cmc.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+VER="3.8"
+LLC="/usr/local/opt/llvm@$VER/bin/llc-$VER"
+CRYPTO="/usr/lib/libcrypto.0.9.8.dylib"
+TEST="$2"
+
+usage() { echo "Usage: $0 [-h help] [-t token] [-a ast] [-l llvm] [-c ll-file] [-s s-file] [-e exe-file] <file-name>.cm" 1>&2; exit 1; }
+
+help() { echo "\n \t Welcome to the C% compiler CMC! 
+				\n \t \t USAGE: $0 [-h help] [-t token] [-a ast] [-l llvm] [-c ll-file] [-s s-file] [-e exe-file] <file-name>.cm\n
```
Comment (joshuazweig) [inline cmc.sh]: Nit picky, but can the tabbing be less aggressive

**Comment by mlm2299** [comment]
@joshuazweig ok if we can't update the path permanently without editing .bash_profile I'm going to cut the bin

**Comment by joshuazweig** [comment]
Yea I think that's fine. Cut it and lets merge

**Comment by mlm2299** [comment]
Should be good to go after travis

**Comment by joshuazweig** [comment]
What I'm seeing here does not have the bin/ 
Can we keep that?

**Comment by mlm2299** [comment]
you just said cut it? -- I don't think the user should have to run ./bin/cmc to run every file

**Comment by joshuazweig** [comment]
I meant cut the path update.

I don't think we should fuck around a bunch in the TLD. This is exactly
what bins are used for all over unix.
If you really don't want to have to run ./bin/cmc put a simlink in the TLD

**Comment by joshuazweig** [comment]
Sorry for the run around/confusion 

Merge it

**Pull request merged**